NLCP/Components/RenderSpritesComponent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Xna.Framework;
+using NLCP.Interfaces;
+using NLCP.Services;
+
+namespace NLCP.Components
+{
+    public class RenderSpritesComponent : DrawableGameComponent
+    {
+        private readonly ISpritesService _spriteService;
+
+        private readonly SpriteBatchService _spriteBatchService;
+
+        public RenderSpritesComponent(Game game) : base(game)
+        {
+            _spriteService = game.Services.GetService<ISpritesService>();
```

Comment:
At some point, when we get tired of repeating `game.Services.GetService` we can consider extension methods